src/map/dto/read-map.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { ResponseSuccessDto } from '../../common/dto/response-success.dto';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ReadMapDto {
```

Comment:
```suggestion
export class ResponseMap {
```

이 어떨까요? 
실질적으로 DTO 로 쓰이는 부분은 아래이고 data 객체에 대한 타입을 명시하는 부분이니 DTO 는 네이밍에서 빠져도 될 것 같습니다.